src/ml/hdcache.hpp

```diff
@@ -8,6 +8,8 @@
 #ifndef __AMS_HDCACHE_HPP__
 #define __AMS_HDCACHE_HPP__
 
+#include <bits/stdc++.h>
```

Comment:
This is GNU specific. Is that desirable?

Reply:
Removed